Mathlib/Algebra/Homology/HomologySequence.lean

```diff
@@ -110,15 +110,10 @@ instance [K.HasHomology i] [K.HasHomology j] :
   dsimp
   infer_instance
 
-#adaptation_note /-- nightly-2024-03-11
-We turn off simprocs here.
-Ideally someone will investigate whether `simp` lemmas can be rearranged
-so that this works without the `set_option`,
-*or* come up with a proposal regarding finer control of disabling simprocs. -/
-set_option simprocs false in
 instance [K.HasHomology i] [K.HasHomology j] :
     Epi ((composableArrows₃ K i j).map' 2 3) := by
-  dsimp
+  -- Disable `Fin.reduceFinMk`, otherwise `Precomp.obj_succ` does not fire.
```

Comment:
Possibly worth opening a mathlib tracking issue about `Precomp.obj_succ` and this simproc interacting badly?